src/Pages/Cars.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Navigation } from 'swiper/modules';
+import { ColorRing } from 'react-loader-spinner';
+import { Link } from 'react-router-dom';
+import CarCard from '../Components/Car/CarCard';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import '../Car.css';
+
+export default function Car() {
```

Comment:
- [ ] Kindly, convert this function to ES6, as requested in the previous review.  

Refactoring to ES6 is easy:

```javascript
const Car= () => {
  // ...
}
```

Don't worry about the export.  You can do it at the end of the file.

That way, your code will comply with the standard. 👍

Reply:
thank you, it totally skipped my mind